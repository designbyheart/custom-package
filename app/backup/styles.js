// @flow
import { StyleSheet, Dimensions, Platform } from 'react-native'
import {
  PADDING_HORIZONTAL,
  PADDING_VERTICAL,
  SHORT_DEVICE,
  color,
  grey,
} from '../common/styles/constant'

const { width, height } = Dimensions.get('window')
const SPACE_FILLER = 50
const amountSpacing = height > SHORT_DEVICE ? PADDING_VERTICAL * 2 : 0
const submitButtonHeight = height > SHORT_DEVICE ? 63 : 43
const showRecoveryButtonHeight = height > SHORT_DEVICE ? 150 : 100
const inputBoxHeight = height > SHORT_DEVICE && Platform.OS === 'ios' ? 137 : 40

const styles = StyleSheet.create({
  tabContainer: {
    maxHeight: 60,
  },
  backgroundImage: {
    flex: 1,
    position: 'absolute',
    transform: [{ rotate: '135deg' }],
  },
  headerCloseIcon: {
    marginRight: PADDING_HORIZONTAL,
    marginBottom: PADDING_VERTICAL,
    alignSelf: 'flex-end',
  },
  headerIcon: {
    marginRight: PADDING_HORIZONTAL,
    marginTop: 15,
    alignSelf: 'flex-end',
  },
  headerBackIcon: {
    marginLeft: PADDING_HORIZONTAL,
    marginTop: 15,
    alignSelf: 'flex-end',
  },
  headerSpacer: {
    height: '100%',
    width: SPACE_FILLER,
  },
  showRecoveryPassphrase: {
    width: '86%',
    height: showRecoveryButtonHeight,
    borderRadius: 5,
    alignSelf: 'center',
    backgroundColor: color.bg.tertiary.color,
    elevation: 7,
    marginBottom: 60,
  },
  genRecovery: {
    backgroundColor: color.bg.eleventh.color,
  },
  genRecoveryWrapper: {
    height: 80,
  },
  genRecoveryHeader: {
    width: '100%',
  },
  genRecoveryPhraseContainer: {
    position: 'absolute',
    top: 20,
    left: 20,
    right: 20,
  },
  genRecoveryPhrase: {
    textAlign: 'center',
    color: color.bg.eleventh.color,
    fontSize: 20,
    lineHeight: 30,
    fontFamily: 'Lato',
    fontWeight: '900',
  },
  genRecoveryText: {
    marginBottom: 20,
    alignSelf: 'center',
    fontSize: 22,
    fontWeight: '600',
    fontFamily: 'Lato',
  },
  genRecoveryMessage: {
    marginBottom: 40,
    paddingHorizontal: 20,
    alignSelf: 'center',
    fontSize: 18,
    lineHeight: 22,
    fontWeight: '500',
    fontFamily: 'Lato',
  },
  genRecoverySecondMessage: {
    marginBottom: 40,
    paddingHorizontal: 20,
    alignSelf: 'center',
    fontSize: 18,
    lineHeight: 22,
    fontWeight: 'bold',
    fontFamily: 'Lato',
  },
  genRecoverySmallMessage: {
    fontSize: 14,
    lineHeight: 17,
    fontWeight: 'bold',
    fontFamily: 'Lato',
    marginBottom: 30,
  },
  verifyMainContainer: {
    flex: 1,
    backgroundColor: color.bg.twelfth.color,
  },
  verifyMainText: {
    paddingHorizontal: 20,
    lineHeight: 22,
  },
  inputBox: {
    marginBottom: 24,
    marginRight: 20,
    marginLeft: 20,
    height: inputBoxHeight,
    borderRadius: 10,
    backgroundColor: 'rgba(0,0,0,0.33)',
    color: 'white',
    padding: 10,
    textAlignVertical: 'top',
  },
  submitButton: {
    marginLeft: '2%',
    marginRight: '2%',
    marginBottom: 15,
    borderRadius: 5,
    shadowColor: grey,
    shadowOffset: {
      width: 1,
      height: 1,
    },
    shadowRadius: 5,
    shadowOpacity: 0.3,
    elevation: 7,
    height: height > SHORT_DEVICE ? 63 : 43,
  },
  backgroundImageVerify: {
    flex: 1,
    position: 'absolute',
  },
  verifyTitle: {
    fontSize: 22,
    marginBottom: 20,
    width: '100%',
  },
  exportBackup: {
    backgroundColor: color.bg.thirteenth.color,
  },
  wrapper: {
    paddingHorizontal: PADDING_HORIZONTAL,
  },
  title: {
    fontSize: 22,
    marginBottom: 20,
    width: '100%',
  },
  exportBackupText: {
    marginBottom: 20,
    backgroundColor: color.bg.thirteenth.color,
    alignSelf: 'center',
    fontSize: 22,
  },
  exportBackupMessage: {
    backgroundColor: color.bg.thirteenth.color,
    alignSelf: 'center',
    fontSize: 18,
  },
  exportBackupSmallMessage: {
    backgroundColor: color.bg.thirteenth.color,
    alignSelf: 'center',
    fontSize: 14,
  },
  backupComplete: {
    backgroundColor: color.bg.fourteenth.color,
  },
  backupCompleteHeader: {
    width: '100%',
  },
  backupCompleteText: {
    marginBottom: 20,
    backgroundColor: color.bg.fourteenth.color,
    alignSelf: 'center',
    fontSize: 22,
  },
  backupCompleteMessage: {
    backgroundColor: color.bg.fourteenth.color,
    alignSelf: 'center',
    fontSize: 18,
  },
  backupCompleteSmallMessage: {
    backgroundColor: color.bg.fourteenth.color,
    alignSelf: 'center',
    fontSize: 14,
  },
  alignItemsCenter: {
    marginBottom: 6,
    marginLeft: '5%',
    marginRight: '5%',
  },
  verticalSpacing: {
    paddingTop: amountSpacing,
  },
  customButtonColor: {
    fontWeight: '600',
    fontSize: 18,
  },
})

export default styles
