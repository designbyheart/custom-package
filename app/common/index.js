// @flow
export * from './api-constants'
export * from './button-constants'
export * from './message-constants'
export * from './route-constants'
export * from './secure-storage-constants'
export function noop() {}
