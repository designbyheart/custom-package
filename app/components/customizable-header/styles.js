import { StyleSheet } from 'react-native'

import { color, font } from '../../common/styles/constant'

export const styles = StyleSheet.create({
  commonHeaderContainer: {
    position: 'absolute',
    width: '100%',
    backgroundColor: 'rgba(255, 255, 255, 0.8)',
  },
  primaryHeaderContainer: {
    height: 120,
    flexDirection: 'row',
    shadowColor: '#000',
    shadowOffset: { width: 0, height: 2 },
    shadowOpacity: 0.2,
    shadowRadius: 10,
  },
  primaryHeaderTextSection: {
    width: '50%',
    height: '100%',
  },
  primaryHeaderIconSection: {
    width: '50%',
    height: '100%',
    flexDirection: 'row-reverse',
  },
  primaryHeaderText: {
    fontFamily: 'Lato',
    fontWeight: 'bold',
    fontSize: 26,
    marginTop: 73,
    marginLeft: 16,
    color: '#505050',
  },
  primaryHeaderIcon: {
    marginTop: 74,
    marginRight: 6,
  },
  connectionDetailsHeaderContainer: {
    height: 136,
  },
  credentialDetailsHeaderContainer: {
    height: 175,
  },
})

export const primaryHeaderContainer = StyleSheet.flatten([
  styles.commonHeaderContainer,
  styles.primaryHeaderContainer,
])

export const connectionDetailsHeaderContainer = StyleSheet.flatten([
  styles.commonHeaderContainer,
  styles.connectionDetailsHeaderContainer,
])

export const credentialDetailsHeaderContainer = StyleSheet.flatten([
  styles.commonHeader,
  styles.credentialDetailsHeaderContainer,
])
