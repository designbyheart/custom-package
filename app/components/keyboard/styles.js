// @flow

import { StyleSheet, Dimensions } from 'react-native'
import { SHORT_DEVICE } from '../../common/styles/constant'
import { PUSH_NOTIFICATION_PERMISSION } from '../../push-notification/type-push-notification'

const { height } = Dimensions.get('window')

const ROW_MARGIN = height > SHORT_DEVICE ? 36 : 15
const FIRST_ROW_MARGIN = height > SHORT_DEVICE ? 24 : 8
const ROW_HEIGHT = height > SHORT_DEVICE ? 33 : 28

const styles = StyleSheet.create({
  container: {
    marginHorizontal: 32, // TODO: update style guide to reflect new constant
  },
  firstRow: {
    height: ROW_HEIGHT,
    marginTop: FIRST_ROW_MARGIN,
  },
  row: {
    height: ROW_HEIGHT,
    marginTop: ROW_MARGIN,
  },
  number: {
    fontSize: 28, // TODO: update style guide to reflect new constant
    fontWeight: '500', // TODO: update style guide to reflect new constant
  },
})

export default styles
