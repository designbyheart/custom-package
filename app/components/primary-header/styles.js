// @flow
import { StyleSheet, Platform, PixelRatio } from 'react-native'

import { isBiggerThanVeryShortDevice } from '../../common/styles/constant'

const containerHeight = isBiggerThanVeryShortDevice ? 100 : 90

// original in icon{} and label{} marginTop was 73 and height was 120, thus the 47 to keep
// subtracting 47 from containerHeight keeps it relative to what is was.
const marginTop = containerHeight - 47

const styles = StyleSheet.create({
  container: {
    position: 'absolute',
    top: 0,
    height: containerHeight,
    width: '100%',
    flexDirection: 'row',
    backgroundColor:
      Platform.OS === 'ios' ? 'rgba(255, 255, 255, 0.8)' : '#fff',
    shadowColor: '#000',
    shadowOffset: { width: 0, height: 2 },
    shadowOpacity: 0.2,
    shadowRadius: 10,
    elevation: Platform.OS === 'android' ? 8 : 0,
  },
  labelSection: {
    width: '75%',
    height: '100%',
  },
  iconsSection: {
    width: '25%',
    height: '100%',
    flexDirection: 'row-reverse',
  },
  label: {
    fontFamily: 'Lato',
    fontWeight: 'bold',
    fontSize: 26,
    marginTop,
    marginLeft: 16,
    color: '#505050',
  },
  icon: {
    marginTop,
    marginRight: 6,
  },
  blur: {
    position: 'absolute',
    left: 0,
    bottom: 0,
    width: '100%',
    height: 120,
  },
})

export { styles }
