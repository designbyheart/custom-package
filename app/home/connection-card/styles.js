// @flow
import { StyleSheet, Platform } from 'react-native'
import { color } from '../../common/styles/constant'

const styles = StyleSheet.create({
  container: {
    width: '100%',
    height: 80,
    flexDirection: 'row',
    borderBottomWidth: 1,
    borderBottomColor: color.border.primary,
  },
  newCardContainer: {
    backgroundColor: color.bg.sixteenth.color,
  },
  avatarSection: {
    height: '100%',
    width: '20%',
    paddingTop: 16,
    paddingLeft: 16,
  },
  infoSection: {
    width: '100%',
    height: '100%',
  },

  infoSectionTopRow: {
    flexDirection: 'row',
    width: '80%',
    height: '50%',
  },
  infoSectionBottomRow: {
    width: '80%',
    height: '50%',
  },
  companyNameSection: {
    width: '70%',
    height: '100%',
    justifyContent: 'flex-end',
  },
  descriptionSection: {
    width: '95%',
    height: '100%',
  },
  dateButtonSection: {
    width: '30%',
    height: '100%',
    flexDirection: 'row',
    alignItems: 'flex-end',
  },
  dateSection: {
    width: '70%',
    height: '100%',
    flexDirection: 'row',
    alignItems: 'flex-end',
    justifyContent: 'flex-end',
  },
  buttonSection: {
    height: '50%',
    width: '30%',
    justifyContent: 'flex-end',
  },
  companyNameText: {
    flex: 1,
    fontFamily: 'Lato',
    fontSize: 17,
    fontWeight: 'bold',
    color: color.textColor.darkgray,
  },
  descriptionText: {
    flex: 1,
    fontFamily: 'Lato',
    fontSize: 14,
    color: color.textColor.grey,
  },
  dateText: {
    fontFamily: 'Lato',
    fontSize: 11,
    color: color.textColor.mediumGray,
    marginBottom: 5,
  },
  newButtonSection: {
    height: '100%',
    width: '30%',
    alignItems: 'center',
    justifyContent: 'flex-end',
  },
  newLabel: {
    width: 64,
    height: 16,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: color.bg.twelfth.color,
    marginBottom: 5,
  },
  newLabelText: {
    fontFamily: 'Lato',
    fontSize: 11,
    fontWeight: '500',
    color: color.bg.primary.font.primary,
  },
})

export { styles }
