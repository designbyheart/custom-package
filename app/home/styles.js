// @flow
import { StyleSheet } from 'react-native'
import { measurements } from '../common/styles/measurements'
import { primaryHeaderHeight } from '../common/styles/constant'

const externalStyles = StyleSheet.create({
  container: {
    width: '100%',
    height: '100%',
    backgroundColor: '#fff',
  },
  flatListContainer: {
    width: '100%',
    height: '100%',
    backgroundColor: '#fff',
    paddingTop: primaryHeaderHeight,
  },
  flatListInnerContainer: {
    paddingBottom: 170,
  },
  blurContainer: {
    position: 'absolute',
    left: 0,
    bottom: 0,
    width: '100%',
    height: measurements.bottomBlurNavBarHeight,
  },
})

export { externalStyles }
