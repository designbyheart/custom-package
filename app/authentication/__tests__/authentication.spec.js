// @flow
import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import { authenticationRoute, PUSH_NOTIFICATION_SENT_CODE } from '../../common/'
import AuthenticationScreen from '../authentication'
import { color } from '../../common/styles'

describe('<AuthenticationScreen />', () => {
  let store = {}

  beforeAll(() => {
    store = {
      getState() {
        return {
          connections: {
            connectionThemes: {
              default: {
                primary: `rgba(${color.actions.button.primary.rgba})`,
                secondary: `rgba(${color.actions.button.secondary.rgba})`,
              },
            },
          },
          route: {
            currentScreen: authenticationRoute,
          },
          authentication: {
            data: {
              offerMsgTitle: 'Hi There',
              offerMsgText:
                'Suncoast Credit Union (sandbox) wants to connect with you',
              statusCode: PUSH_NOTIFICATION_SENT_CODE,
              logoUrl: null,
              remoteConnectionId: 'B4Y9fhpeHdGHBKKtSgAYrB',
            },
          },
        }
      },
      subscribe() {
        return jest.fn()
      },
      dispatch() {
        return jest.fn()
      },
    }
  })

  it('should match snapshot', () => {
    const component = renderer.create(
      <Provider store={store}>
        <AuthenticationScreen />
      </Provider>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
