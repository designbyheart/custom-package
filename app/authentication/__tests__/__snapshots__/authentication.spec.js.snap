// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthenticationScreen /> should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      fifth={true}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
        }
      }
    >
      <View
        hCenter={true}
        style={
          Object {
            "flex": 1,
            "justifyContent": "center",
          }
        }
        testID="invitation-text-container"
        useNativeDriver={true}
      >
        <View
          bottom={true}
          style={
            Object {
              "flex": 1,
              "justifyContent": "flex-end",
            }
          }
          testID="invitation-text-message-container"
        >
          <Animatable.View
            animation="bounceInDown"
            delay={100}
            duration={1500}
            testID="invitation-text-container-animation"
          >
            <View
              center={true}
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                }
              }
              testID="invitation-text-container-message-title"
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                onPress={[Function]}
                style={
                  Array [
                    Object {
                      "fontSize": 20,
                    },
                    Object {
                      "fontFamily": "SF Pro Display",
                    },
                    Object {
                      "backgroundColor": "#ffffff",
                      "color": "#777777",
                    },
                    Object {
                      "fontWeight": "800",
                    },
                    Object {
                      "textAlign": "center",
                    },
                    null,
                    Object {
                      "lineHeight": 30,
                      "marginHorizontal": 30,
                      "marginVertical": 10,
                    },
                  ]
                }
                testID="invitation-text-container-title"
              >
                Hi There
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                onPress={undefined}
                style={
                  Array [
                    Object {
                      "fontSize": 17,
                    },
                    Object {
                      "fontFamily": "SF Pro Display",
                    },
                    Object {
                      "backgroundColor": "#ffffff",
                      "color": "#777777",
                    },
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "textAlign": "center",
                    },
                    null,
                    Object {
                      "margin": 10,
                    },
                  ]
                }
                testID="invitation-text-container-message"
              >
                Suncoast Credit Union (sandbox) wants to connect with you
              </Text>
            </View>
          </Animatable.View>
        </View>
        <View
          style={
            Object {
              "flex": 1,
            }
          }
          testID="invitation-text-container-avatars"
        >
          <Animatable.View
            animation="bounceInUp"
            delay={300}
            duration={1500}
            testID="invitation-text-container-avatars-animation"
          >
            <View
              center={true}
              spaceBetween={true}
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "space-between",
                }
              }
            >
              <View
                row={true}
                spaceBetween={true}
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "space-between",
                    "marginVertical": 40,
                  }
                }
                testID="invitation-text-avatars-container"
                vCenter={true}
              >
                <View
                  imageShadow="imageShadow"
                  style={
                    Object {
                      "borderRadius": 38,
                      "shadowColor": "rgba(0, 0, 0, 0.25)",
                      "shadowOffset": Object {
                        "height": 3,
                        "width": 0,
                      },
                      "shadowOpacity": 1,
                      "shadowRadius": 8,
                    }
                  }
                  testID="invitation-text-avatars-invitee"
                >
                  <Image
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    onError={undefined}
                    onLayout={undefined}
                    onLoad={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    source={1}
                    style={
                      Array [
                        Object {
                          "height": 76,
                          "width": 76,
                        },
                        Object {
                          "borderRadius": 38,
                        },
                      ]
                    }
                    testID={undefined}
                  />
                </View>
                <Image
                  source={1}
                  style={
                    Object {
                      "height": 8,
                      "marginHorizontal": 5,
                      "width": 60,
                    }
                  }
                />
                <View
                  imageShadow="imageShadow"
                  style={
                    Object {
                      "borderRadius": 38,
                      "shadowColor": "rgba(0, 0, 0, 0.25)",
                      "shadowOffset": Object {
                        "height": 3,
                        "width": 0,
                      },
                      "shadowOpacity": 1,
                      "shadowRadius": 8,
                    }
                  }
                  testID="invitation-text-avatars-inviter"
                >
                  <Image
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    onError={undefined}
                    onLayout={undefined}
                    onLoad={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    source={1}
                    style={
                      Array [
                        Object {
                          "height": 76,
                          "width": 76,
                        },
                        Object {
                          "borderRadius": 38,
                        },
                      ]
                    }
                    testID={undefined}
                  />
                </View>
              </View>
            </View>
          </Animatable.View>
        </View>
      </View>
    </View>
    <View>
      <View>
        <View
          row={true}
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              style={Object {}}
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                    },
                    undefined,
                  ]
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <View
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#9E9E9E",
                        "flexDirection": "row",
                        "justifyContent": "center",
                        "marginLeft": 15,
                        "marginRight": 15,
                        "padding": 19,
                      },
                      Object {
                        "backgroundColor": "#8F0CE8",
                      },
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      Object {
                        "backgroundColor": "rgba(133, 191, 67, 0.5)",
                      },
                      undefined,
                      Object {
                        "shadowColor": "rgba(0,0,0, .4)",
                        "shadowOffset": Object {
                          "height": 1,
                          "width": 1,
                        },
                        "shadowOpacity": 1,
                        "shadowRadius": 1,
                      },
                      Object {
                        "padding": 12,
                      },
                      Array [
                        Object {
                          "marginLeft": 0,
                          "marginRight": 0,
                        },
                        Object {
                          "paddingVertical": 17,
                        },
                        null,
                      ],
                      undefined,
                      undefined,
                    ]
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Array [
                        Object {},
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        Array [
                          Object {
                            "color": "white",
                            "fontSize": 16,
                          },
                          undefined,
                          Object {
                            "fontSize": 14,
                          },
                          undefined,
                          Object {
                            "fontSize": 17,
                            "fontWeight": "600",
                          },
                          undefined,
                          undefined,
                        ],
                      ]
                    }
                  >
                    Deny
                  </Text>
                </View>
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          >
            <View
              style={Object {}}
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                hasTVPreferredFocus={undefined}
                hitSlop={undefined}
                isTVSelectable={true}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                    },
                    undefined,
                  ]
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <View
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#9E9E9E",
                        "flexDirection": "row",
                        "justifyContent": "center",
                        "marginLeft": 15,
                        "marginRight": 15,
                        "padding": 19,
                      },
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      undefined,
                      Object {
                        "backgroundColor": "rgba(133, 191, 67, 1.0)",
                      },
                      undefined,
                      Object {
                        "shadowColor": "rgba(0,0,0, .4)",
                        "shadowOffset": Object {
                          "height": 1,
                          "width": 1,
                        },
                        "shadowOpacity": 1,
                        "shadowRadius": 1,
                      },
                      Object {
                        "padding": 12,
                      },
                      Array [
                        Object {
                          "marginLeft": 0,
                          "marginRight": 0,
                        },
                        Object {
                          "paddingVertical": 17,
                        },
                        null,
                      ],
                      undefined,
                      undefined,
                    ]
                  }
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Array [
                        Object {},
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        undefined,
                        Array [
                          Object {
                            "color": "white",
                            "fontSize": 16,
                          },
                          undefined,
                          Object {
                            "fontSize": 14,
                          },
                          undefined,
                          Object {
                            "fontSize": 17,
                            "fontWeight": "600",
                          },
                          undefined,
                          undefined,
                        ],
                      ]
                    }
                  >
                    Connect
                  </Text>
                </View>
              </View>
            </View>
          </View>
        </View>
        <WebView
          javaScriptEnabled={true}
          onMessage={[Function]}
          source={
            Object {
              "html": "
                    <!DOCTYPE html>
                    <html>
                    <script>
                    function fetchColorFromCanvas(){
                      var rgb = getAverageRGB(document.getElementById(\\"canvasImage\\"));
                      var message = {\\"message\\":'rgbValues',\\"payload\\":rgb};
                      window.postMessage(JSON.stringify(message));
                    }
            
                    function getAverageRGB(imgEl) {
                      var blockSize = 5;
                      var defaultRGB = {r:0,g:0,b:0};
                      var canvas = document.createElement(\\"canvas\\");
                      var context = canvas.getContext && canvas.getContext(\\"2d\\");
                      var data, width, height;
                      var i = -4;
                      var length;
                      var rgb = {r:0,g:0,b:0};
                      var count = 0;
                      if (context == undefined || context == null) {
                          return defaultRGB;
                      }
                      height = canvas.height = imgEl.naturalHeight || imgEl.offsetHeight || imgEl.height;
                      width = canvas.width = imgEl.naturalWidth || imgEl.offsetWidth || imgEl.width;
                      context.drawImage(imgEl, 0, 0);
                      try {
                          data = context.getImageData(width/3, width/3 + 10, width/3, width/3 + 10);
                      } catch(e) {
                          return defaultRGB;
                      }
            
                      length = data.data.length;  
                      while ( (i += blockSize * 4) < length ) {
                          ++count;
                          rgb.r += data.data[i];
                          rgb.g += data.data[i+1];
                          rgb.b += data.data[i+2];              
                      }
                      rgb.r = ~~(rgb.r/count);
                      rgb.g = ~~(rgb.g/count);
                      rgb.b = ~~(rgb.b/count);
                      return rgb;
                    }
            
                    const interval = setInterval(function() {
                      var img = document.getElementById('canvasImage');
                      if(img.src.length != 0) {
                        fetchColorFromCanvas();
                        clearInterval(interval);
                      }
                    },10);
            
                    </script>
                    <body style='position: absolute; bottom: -1000px;'>
                      <img src='data:image/jpeg;base64,' id='canvasImage' onload='fetchColorFromCanvas()'/>
                    </body>
                    </html>",
            }
          }
          style={
            Object {
              "position": "absolute",
              "right": -1000,
            }
          }
        />
      </View>
    </View>
  </View>
  <Modal
    backdropColor="#f0f0f0"
    backdropOpacity={1}
    isVisible={false}
  >
    <View
      fifth={true}
      shadow={true}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "marginHorizontal": 30,
          "shadowColor": "rgba(0, 0, 0, 0.2)",
          "shadowOffset": Object {
            "height": 10,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 30,
        }
      }
    >
      <View
        center={true}
        style={
          Object {
            "alignItems": "center",
            "borderBottomColor": "#3f3f3f",
            "borderBottomWidth": NaN,
            "justifyContent": "center",
            "paddingVertical": 10,
          }
        }
      >
        <View
          row={true}
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 10,
              "width": 220,
            }
          }
          testID="invitation-text-avatars-container"
          vCenter={true}
        >
          <View
            imageShadow="imageShadow"
            style={
              Object {
                "borderRadius": 38,
                "shadowColor": "rgba(0, 0, 0, 0.25)",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 1,
                "shadowRadius": 8,
              }
            }
            testID="invitation-text-avatars-invitee"
          >
            <Image
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onError={undefined}
              onLayout={undefined}
              onLoad={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              source={1}
              style={
                Array [
                  Object {
                    "height": 76,
                    "width": 76,
                  },
                  Object {
                    "borderRadius": 38,
                  },
                ]
              }
              testID={undefined}
            />
          </View>
          <Image
            source={1}
            style={
              Object {
                "height": 22,
                "marginHorizontal": 20,
                "width": 30,
              }
            }
          />
          <View
            imageShadow="imageShadow"
            style={
              Object {
                "borderRadius": 38,
                "shadowColor": "rgba(0, 0, 0, 0.25)",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 1,
                "shadowRadius": 8,
              }
            }
            testID="invitation-text-avatars-inviter"
          >
            <Image
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hitSlop={undefined}
              onError={undefined}
              onLayout={undefined}
              onLoad={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              source={1}
              style={
                Array [
                  Object {
                    "height": 76,
                    "width": 76,
                  },
                  Object {
                    "borderRadius": 38,
                  },
                ]
              }
              testID={undefined}
            />
          </View>
        </View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={undefined}
          style={
            Array [
              Object {
                "fontSize": 17,
              },
              Object {
                "fontFamily": "SF Pro Display",
              },
              Object {
                "backgroundColor": "#ffffff",
                "color": "#777777",
              },
              Object {
                "fontWeight": "600",
              },
              Object {
                "textAlign": "center",
              },
              null,
              Object {
                "color": "#777777",
                "margin": 10,
                "marginHorizontal": 30,
              },
            ]
          }
          testID="modal-body"
        >
          You are now connected to 
          Evernym
          !
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#9E9E9E",
                "flexDirection": "row",
                "justifyContent": "center",
                "marginLeft": 15,
                "marginRight": 15,
                "padding": 19,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Object {
                "backgroundColor": "#ffffff",
              },
              undefined,
              undefined,
              Object {
                "padding": 12,
              },
              Array [
                Object {
                  "marginLeft": 0,
                  "marginRight": 0,
                },
                null,
                null,
              ],
              undefined,
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 16,
                  },
                  Object {
                    "color": "#85bf43",
                  },
                  Object {
                    "fontSize": 14,
                  },
                  undefined,
                  Object {
                    "fontSize": 17,
                    "fontWeight": "600",
                  },
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Continue
          </Text>
        </View>
      </View>
    </View>
  </Modal>
</View>
`;
